packages/cli/test/cli.spec.ts

```diff
@@ -184,6 +184,72 @@ describe('Stylable Cli', () => {
         ).equal(true);
     });
 
+    it('build .st.css.d.ts alongside source files with source-maps on by default', () => {
+        const srcContent = '.root{color:red}';
+        populateDirectorySync(tempDir.path, {
+            'package.json': `{"name": "test", "version": "0.0.0"}`,
+            'style.st.css': srcContent,
+        });
+
+        const { stdout, stderr } = runCli([
+            '--rootDir',
+            tempDir.path,
+            '--outDir',
+            'dist',
+            '--stcss',
+            '--dts',
+        ]);
+
+        expect(stderr).equal('');
+        expect(stdout).equal('');
+
+        const dirContent = loadDirSync(tempDir.path);
+        const stylesheetContent = dirContent[join('dist', 'style.st.css')];
+        const dtsContent = dirContent[join('dist', 'style.st.css.d.ts')];
+        const dtsSourceMapContent = dirContent[join('dist', 'style.st.css.d.ts.map')];
+
+        expect(stylesheetContent).to.equal(srcContent);
+        expect(dtsContent.startsWith('/* THIS FILE IS AUTO GENERATED DO NOT MODIFY */')).to.equal(
+            true
+        );
+        expect(
+            dtsSourceMapContent.startsWith('{\n    "version": 3,\n    "file": "style.st.css.d.ts"')
```

Comment:
not sure that the version in important to check here.

Reply:
Just checking the start of the sourcemap, the version here is for the sourcemap syntax.